Runtime/CesiumSubScene.cs

```diff
@@ -310,13 +347,7 @@ private void OnParentChanged()
 
             // Update our origin to our parent georef, maintain our origin authority,
             // and copy both sets of reference coordinates. No need to calculate any of this again
-            this._longitude = this._parentGeoreference.longitude;
-            this._latitude = this._parentGeoreference.latitude;
-            this._height = this._parentGeoreference.height;
-
-            this._ecefX = this._parentGeoreference.ecefX;
-            this._ecefY = this._parentGeoreference.ecefY;
-            this._ecefZ = this._parentGeoreference.ecefZ;
+            CopyParentCoordinates();
```

Comment:
Just a nitpick for stylistic consistency.

```suggestion
            this.CopyParentCoordinates();
```